src/DataObjectAnnotator.php

```diff
@@ -168,18 +261,18 @@ protected function writeFileContent($className)
         $filePath = $classInfo->getClassFilePath();
 
         if (!is_writable($filePath)) {
+            // Unsure how to test this properly
```

Comment:
Ah the issue of direct output in build tasks (and related operations) 😆 you can test this in one of two ways:

* Buffer the output of the test
* Move the call to `DB::alterati...` into a class method, then partially mock it

Related: https://github.com/silverstripe/silverstripe-framework/issues/6524

Reply:
It's the make-path-unwritable-mid-operation is what I'm referring to ;)